src/types/color.ts

```diff
@@ -0,0 +1,7 @@
+export enum Color {
```

Comment:
Diría de hacer un tipo con literales en vez de enum, por lo que comenté más abajo en el tsconfig

```suggestion
export type TeamColor = "Rojo" | "Azul" | ...
```